src/test/java/org/openmrs/contrib/qaframework/automation/ConditionsSteps.java

```diff
@@ -8,14 +8,17 @@
  * graphic logo is a trademark of OpenMRS Inc.
  */
 package org.openmrs.contrib.qaframework.automation;
-
+ 
 import static org.junit.Assert.assertNotNull;
 import static org.junit.Assert.assertNull;
 
 import org.apache.commons.lang3.StringUtils;
 import org.openmrs.contrib.qaframework.RunTest;
+import org.openmrs.contrib.qaframework.helper.TestData;
+import org.openmrs.contrib.qaframework.page.ClinicianFacingPatientDashboardPage;
```

Comment:
do you need this import?
```suggestion
import org.openmrs.contrib.qaframework.page.ClinicianFacingPatientDashboardPage;
```

Reply:
it's used here -> https://github.com/openmrs/openmrs-contrib-qaframework/blob/71b4fbe3139b5ffb16e9710d0a9c7a9b7335bb76/src/test/java/org/openmrs/contrib/qaframework/automation/ConditionsSteps.java#L43